_people/Sabrina_Shams.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Sabrina Shams
+position: underGradStudent
+avatar: Sabrina_Shams.jpg
+twitter:
+joined: 2020
+---
+
+<img width="300" src="{{site.baseurl}}/images/people/{{page.avatar}}" data-action="zoom">
+
```

Comment:
Great work, Sabrina!
I would recommend adding a Contact section with your ufl email and an About section with some information about you. It might be helpful to look at my code, which you can access in this repository by going to the _people folder, clicking my file, and selecting "Raw".